server/test/categoryRouter.test.js

```diff
@@ -1,5 +1,6 @@
 const Category = require('../db/models/category');
-const categoryRouter = require('../api/categoryRouter');
+// const categoryRouter = require('../api/categoryRouter');
+const server = require('../index');
```

Comment:
@Anthony-Fiddes

Reply:
this allows us to not do router.use(express.json) every time

Reply:
I love her!